cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -214,6 +235,122 @@ public CohortEngineRestHandler() {
 		// NOTE: This constructor is called on every REST call!
 	}
 
+	@POST
+	@Path("/cohort-evaluation")
+	@Consumes(MediaType.MULTIPART_FORM_DATA)
+	@Produces({ MediaType.APPLICATION_JSON })
+	@ApiOperation(value = "Evaluates a specific define within a CQL for a set of patients"
+			, notes = EVALUATION_API_NOTES, response = String.class
+			, tags = {"Cohort Evaluation"}
+			, nickname = "evaluate_cohort"
+			, extensions = {
+			@Extension(properties = {
+					@ExtensionProperty(
+							name = DarkFeatureSwaggerFilter.DARK_FEATURE_NAME
+							, value = CohortEngineRestConstants.DARK_LAUNCHED_COHORT_EVALUATION)
+			})
+	}
+	)
+	@ApiImplicitParams({
+			// This is necessary for the dark launch feature
+			@ApiImplicitParam(access = DarkFeatureSwaggerFilter.DARK_FEATURE_CONTROLLED, paramType = "header", dataType = "string"),
+			// These are necessary to create a proper view of the request body that is all wrapped up in the Liberty IMultipartBody parameter
+			@ApiImplicitParam(name=REQUEST_DATA_PART, value=EXAMPLE_REQUEST_DATA_JSON, dataTypeClass = MeasureEvaluation.class, required=true, paramType="form", type="file"),
+			@ApiImplicitParam(name=CQL_PART, dataTypeClass = File.class, required=true, paramType="form", type="file" )
+	})
+	@ApiResponses(value = {
+			@ApiResponse(code = 200, message = "Successful Operation"),
+			@ApiResponse(code = 400, message = "Bad Request", response = ServiceErrorList.class),
+			@ApiResponse(code = 500, message = "Server Error", response = ServiceErrorList.class)
+	})
+	public Response evaluateCohort(@Context HttpServletRequest request,
+
+									@ApiParam(value = ServiceBaseConstants.MINOR_VERSION_DESCRIPTION, required = true, defaultValue = ServiceBuildConstants.DATE) @QueryParam(CohortEngineRestHandler.VERSION) String version,
+								   @ApiParam(value = CQL_DEFINE, required = true) @QueryParam(CohortEngineRestHandler.COHORT_DEFINE) String defineToRun,
+								   @ApiParam(value = PATIENT_IDS, required = true) @QueryParam(CohortEngineRestHandler.COHORT_PATIENT_ID) String patientIds,
+								   @ApiParam(value = CohortEngineRestHandler.COHORT_ENABLE_LOGGING, defaultValue = "false") @DefaultValue ("NA") @QueryParam(CohortEngineRestHandler.ENABLE_LOGGING) LoggingEnum enableLogging,
+								   @ApiParam(hidden = true, type="file", required=true) IMultipartBody multipartBody)
```

Comment:
Pedantic: The above four lines seem to have spaces mingled with tabs.

Reply:
fixed
\